parlai/mturk/core/dev/agents.py

```diff
@@ -515,24 +517,13 @@ def wait_completion_timeout(self, iterations):
             i += 1
         return
 
-    # TODO clean up this whole thing once completes are guaranteed posts
+    # TODO cleanup timeout now that it's not used.
     def wait_for_hit_completion(self, timeout=None):
         """Waits for a hit to be marked as complete"""
-        # Timeout in seconds, after which the HIT will be expired automatically
-        if timeout:
-            if timeout < 0:
-                # Negative timeout is for testing, wait for packet to send
-                time.sleep(1)
-                self.m_free_workers([self])
-                return True
-            start_time = time.time()
         wait_periods = 1
         self.wait_completion_timeout(wait_periods)
         sync_attempts = 0
-        while (
-            not self.hit_is_complete
-            and self.m_get_agent_work_status(self.assignment_id) != self.ASSIGNMENT_DONE
-        ):
+        while not self.hit_is_complete:
             if sync_attempts < 8:
```

Comment:
magic constant

Reply:
Turns out we don't need most of this block anymore